package.json

```diff
@@ -47,9 +47,11 @@
     "typescript": "^4.2.4"
   },
   "dependencies": {
+    "@grpc/grpc-js": "^1.3.5",
     "@newrelic/telemetry-sdk": "^0.4.2",
     "@opentelemetry/api": "^1.0.1",
     "@opentelemetry/exporter-collector": "^0.23.0",
```

Comment:
Is this still required?

Reply:
grpc-js is required for sending the API key with the request.

Reply:
...though now, having worked with the simpler example, I think this explicit install needs to be removed.